src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/IndexRecommendationTests.cs

```diff
@@ -30,14 +31,22 @@ public IndexRecommendationTests(ITestOutputHelper output) : base(output)
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestGetIndexRecommendation()
         {
-            RunPowerShellTest("Test-GetIndexRecommendations");
+            // Test cannot be re-recorded because it has hardcoded server name
+            if (HttpMockServer.Mode == HttpRecorderMode.Playback)
```

Comment:
Discussed offline, it is only the server resource api-version being changed, not other resources.